.github/workflows/build.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       GIT_TAG: ${{ steps.variables.outputs.GIT_TAG }}
       GIT_BRANCH: ${{ steps.variables.outputs.GIT_BRANCH }}
       OUTPUT_DIR: ${{ steps.variables.outputs.OUTPUT_DIR }}
+      DO_DEPLOY: ${{ github.event_name != 'pull_request' && inputs.SSH_KEY != '' && github.actor != 'dependabot[bot]' }}
```

Comment:
Not sure if this is working. All examples I could find use `>> $GITHUB_OUTPUT` in a step that exports output.